e2e/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
could add a description for funzies

Reply:
I can add one in, do you know what it's used for/ where it shows up?

Reply:
We called it `Calendar e2e` to match the description in the package.json file in api (`Calendar API`)

Reply:
You could also say something descriptive like `end to end testing with cypress for calendar app`

Reply:
I updated it with that description ✅